integrations_test.go

```diff
@@ -1,17 +1,24 @@
 package rockset_test
 
 import (
+	"context"
 	"errors"
 	"log"
 	"os"
 	"testing"
 
+	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 
 	"github.com/rockset/rockset-go-client"
 	"github.com/rockset/rockset-go-client/option"
 )
 
+func (s *IntegrationsSuite) TestAzureBlob() {
+	s.T().Skip("pending better abstraction for local testing")
```

Comment:
We need an Integration test for this, like we have for e.g. the S3 integration. I can help you get Azure credentials injected into the Circle CI environment.

Reply:
Filled the test out much like the S3 test, with obvious changes and use of the test suite